### PR TITLE
fix metric bugs for lower, support mindir export with model yaml config

### PR DESCRIPTION
**Pull request by SamitHuang**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: tools/export.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
     >>> python tools/export.py --model_name dbnet_resnet50 --data_shape 736 1280
     >>> # Export mindir of model `dbnet_resnet50` by loading local ckpt
     >>> python tools/export.py --model_name dbnet_resnet50 --data_shape 736 1280 --local_ckpt_path /path/to/local_ckpt
+    >>> # Export mindir of model whose architecture is defined by crnn_resnet34.yaml with local checkpoint
```
Comment (HaoyangLee) [inline tools/export.py:15]: can also update the export usage here: https://github.com/mindspore-lab/mindocr/tree/main/configs#model-export

**Review by HaoyangLee — APPROVED** [review]

**Comment by zhtmike** [comment]
如果是yaml配置，因为没有预设checkpoint url，建议 必须提供local checkpoint path 作为保护

File: tools/export.py (hunk 1 above)
Comment (SamitHuang) [reply tools/export.py:15]: updated

**Comment by SamitHuang** [comment]
> 如果是yaml配置，因为没有预设checkpoint url，建议 必须提供local checkpoint path 作为保护
added

**Review by zhtmike — APPROVED** [review]

**Pull request merged**